src/test/testFiles/valid/binaryTreeReturn.manimdsl

```diff
@@ -0,0 +1,8 @@
+fun test(): Tree<Node<number>> {
```

Comment:
not sure if we want to include this example since the animation for returning trees doesn't really work?